src/instruments/src/Clock/Components/DayNight.tsx

```diff
@@ -0,0 +1,7 @@
+function getDayNightState(seconds: number) {
+    const localHour = Math.floor(seconds / 3600);
+    const timeOfDay = (localHour >= 6 && localHour <= 18) ? 'day' : 'night';
+    return timeOfDay;
+}
+
+export default getDayNightState;
```

Comment:
While this is better, with it being in a Components folder this could still be mistaken for a component. You could move it to a separate file in the Clock directory and give it some kind of more generic filename, like `utils` or `helpers`.